test-suite-http-server-tck-azure-function-http/src/test/java/io/micronaut/http/server/tck/azurehttpfunction/HttpRequestMessageAdapter.java

```diff
@@ -0,0 +1,47 @@
+package io.micronaut.http.server.tck.azurehttpfunction;
+
+import com.microsoft.azure.functions.HttpMethod;
+import com.microsoft.azure.functions.HttpRequestMessage;
+import com.microsoft.azure.functions.HttpResponseMessage;
+import com.microsoft.azure.functions.HttpStatus;
+import com.microsoft.azure.functions.HttpStatusType;
+
+import java.net.URI;
+import java.util.Map;
+
+public class HttpRequestMessageAdapter implements HttpRequestMessage {
+    @Override
+    public URI getUri() {
+        return null;
```

Comment:
is this normal

Reply:
I'm not sure...  You added that on 19th December... Was I supposed to alter it?